org/party.go

```diff
@@ -101,6 +103,18 @@ type Telephone struct {
 	Number string `json:"num" jsonschema:"title=Number"`
 }
 
+// Website describes what is expected for a web address.
+type Website struct {
+	// Unique identity code
+	UUID *uuid.UUID `json:"uuid,omitempty" jsonschema:"title=UUID"`
+	// Identifier for this number.
+	Label string `json:"label,omitempty" jsonschema:"title=Label"`
+	// Title of the website to help distinguish between this and other links.
+	Title string `json:"title,omitempty" jsonschema:"title=Title"`
+	// URL for the website.
+	URL string `json:"url" jsonschema:"title=URL,format=uri"`
```

Comment:
Property name could be `URI` instead of `URL`, but I think its okay in this case to more precisely indicate "Location".